src_features/signTx/logic_signTx.c

```diff
@@ -257,22 +257,6 @@ void prepareFeeDisplay() {
                        sizeof(strings.common.maxFee));
 }
 
-uint32_t get_chainID() {
```

Comment:
this fn was not used... there's a second `get_chainid()` that is actually used ^^